shimming-toolbox/shimmingtoolbox/cli/maths.py

```diff
@@ -61,6 +61,78 @@ def mean(fname_input, fname_output, axis, verbose):
     nib.save(nii_output, fname_output)
 
 
+@maths_cli.command(context_settings=CONTEXT_SETTINGS)
+@click.option('-i', '--input', 'fname_input', type=click.Path(exists=True), required=True,
+              help="Input filename, supported extensions: .nii, .nii.gz")
+@click.option('-o', '--output', 'fname_output', type=click.Path(), default=None,
+              help="Output filename, supported extensions: .nii, .nii.gz. [default: ./input_std.nii.gz]")
+@click.option('--axis', type=click.Choice(AXES), default=AXES[3], show_default=True,
+              help="Axis of the array to calculate the average")
```

Comment:
I don't see the point of limiting the axis. I would simply require an integer between -1 and nb_dims. -1 should be the default value.

Reply:
The reason is that it is easier for a user to see what the input data can be at a glance using `click.Choice` compared to `click.IntRange`. However, I do agree that having -1 as a default option is better, so I added the option as well.